vendor/loader-shim.js

```diff
@@ -0,0 +1,10 @@
+/* globals define */
+define('ember/load-initializers', ['exports', 'ember-load-initializers', 'ember'], function(exports, loadInitializers, Ember) {
```

Comment:
I suppose i included it to be thorough in including all of ember-load-initializers in the (inner) app. But I guess it is not necessary.